src/loaders/ObjectLoader.js

```diff
@@ -737,6 +810,10 @@ Object.assign( ObjectLoader.prototype, {
 
 				}
 
```

Comment:
Is there a `break;` missing here for the Mesh case?

Reply:
Oops, thanks. I'll fix later.